.github/workflows/codespell.yaml

```diff
@@ -0,0 +1,23 @@
+name: Codespell
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+  pull_request:
+    branches:
+      - "master"
+jobs:
+  codespell:
+    name: codespell
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: codespell
+        uses: codespell-project/actions-codespell@master
+        with:
+          skip: .git,*.png,*.jpg
+          ignore_words_list: aks,keyserver,atleast,dne,ser,ist,files',ba,dum,iam,te
```

Comment:
Do you have to increment this every time something new is not recognized?

Reply:
Yes.

Reply:
We would be able to use `ignore_words_files` here. However, I think listing words here is enough so far.

Reply:
In next release of codespell there will be config file available, which then will be preferable way of defining those I think.

Reply:
- Do we need all these in the ignore list? Seems like we could fix some of them like "at least" or reword `files'` so we don't need the apostrophe. 
- Are these the only ignore words we need for a clean run? I'm surprised how short the list is.

Reply:
>Do we need all these in the ignore list? Seems like we could fix some of them like "at least" or reword files' so we don't need the apostrophe.

`atleast` is a method name. There is also `IsAtLeast`, so I think it's reasonable to ignore that. `files'` could be reworded, but I didn't want to increase the scope of the PR. 

>Are these the only ignore words we need for a clean run? I'm surprised how short the list is.

Well, workflows are passing here, soo yes. Do note, that this PR is a follow up to #6563, where I fixed all fixable typos :smile:

Reply:
Sounds good, and thanks for fixing all these issues! It's always a good feeling to improve the code hygiene.